docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -87,6 +85,13 @@ and postponed until the current task is completed.
   }
  }
 
+:group_call_report:`_` ::
+
+ {
+     "_href": "/pulp/api/v2/task_groups/16412fcb-06fa-4caa-818b-b103e2a9bf44/state_summary/",
```

Comment:
How about making a 1-line view function for the root of the resource that raises 404, but then you at least have something to reference with ``reverse`` ?